.travis.yml

```diff
@@ -80,16 +89,15 @@ jobs:
           skip_cleanup: true
           skip_upload_docs: true
           on:
-            branch: dev
-            repo: plus3it/pyppyn
+            branch: $RELEASE_BRANCH
     - stage: deploy
-      if: branch = master AND repo = plus3it/pyppyn
+      if: branch = env(RELEASE_BRANCH) AND tag IS blank
```

Comment:
`if: type != pull_request`

Reply:
i made it `if: branch = env(RELEASE_BRANCH) AND type != pull_request` so that it wouldn't try to deploy on my repo's feature branch